renderdoc/driver/gl/gl_replay.cpp

```diff
@@ -3232,6 +3276,13 @@ bool GLReplay::IsTextureSupported(const ResourceFormat &format)
   return true;
 }
 
+bool GLReplay::NeedRemapForFetch(const ResourceFormat &format)
+{
+  if(format.compType == CompType::Depth)
+    return IsGLES && !HasExt[NV_read_depth];
```

Comment:
You might want to check here to see if `format.specialType` is one of the combined depth/stencil formats. I think I set this `CompType::Depth` pretty consistently, but it would be good to be bullet-proof just in case.